ElectronClient/gui/NoteText.jsx

```diff
@@ -1612,18 +1612,30 @@ class NoteTextComponent extends React.Component {
 		menu.popup(bridge().window());
 	}
 
+	async updateFolderTreeOnSearch(folder) {
+		while (folder && folder.parent_id) {
+			this.props.dispatch({
+				type: 'FOLDER_TOGGLE',
```

Comment:
You toggle the folder - doesn't this mean that if the folder is already open you're going to close it?

By the way this logic is already in GotoAnything.jsx. Could you extract it from there, move it to models/Folder and reuse it here?

Reply:
> You toggle the folder - doesn't this mean that if the folder is already open you're going to close it?

I didn't think about that. Will fix it.